utils/_context/_scenarios.py

```diff
@@ -67,7 +67,7 @@ def session_start(self, session):
         self.terminal = session.config.pluginmanager.get_plugin("terminalreporter")
         self.terminal.write_sep("=", "test context", bold=True)
         self.print_info(f"Scenario: {self.name}")
-        self.print_info(f"Logs folder: ./{self.host_log_folder}")
+        self.print_info(f"Logs folder: {self.host_log_folder}")
```

Comment:
The `./` is quite important for devx, as it make the item clickable on most bash terminal

Reply:
Alright. Would it work if it were an absolute path as well?